tests/profile/hmmer/CPtrStore.csv

```diff
@@ -0,0 +1,255 @@
+DigitizeAlignment_call5	DigitizeAlignment_arrayidx
```

Comment:
Okay, for some reason these files are explicitly documented as csv, the .csv extension is hardcoded in, and heaps of functions reference csv in their function or variable names, so I can't just fix those files, this would be a big change that would break backwards compatibility. It'd require less changes to change the tab separators to commas, but I assume there's some non-obvious reason for it not working that way now, plus that'd break compatibility too. This would need to be tackled as it's own issue if it's something we care about.